jest.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: ['src/filters/**/*.js'],
+  collectCoverageFrom: ['src/filters/**/*.js', '!src/filters/images.js'],
```

Comment:
I'm torn on whether or not we should test the new `image` shortcode. I think we'd have to mock/stub out the `@11ty/eleventy-img` package.

Reply:
> 
> 
> I'm torn on whether or not we should test the new `image` shortcode. I think we'd have to mock/stub out the `@11ty/eleventy-img` package.

@jgerigmeyer Is there some place to look for an example of making a mock/stub of a package like this for testing?

Reply:
@dvdherron Sorry, I started on these tests last evening and just saw this comment. See https://github.com/oddbird/oddleventy/commit/4754158d34fc555845113d1c2204fe03420f17b8